templates/index.html

```diff
@@ -16,8 +16,15 @@ <h1>
       <p>
         Automate deployment and operations of network functions, and benefit from reduced operational costs, by using an open source implementation of ETSI NFV MANO. To improve the stability of deployments and provide telcos with a production-grade platform, Canonical delivers a pure upstream OSM distribution &mdash; Charmed OSM. As a key contributor to the Open Source MANO project, we are excited to help you on your NFV journey.
       </p>
-      <p><a href="/contact-us" class="p-button--positive">Get in touch</a></p>
-      <p><a href="https://tutorials.ubuntu.com/tutorial/charmed-osm-get-started" class="p-link--inverted p-link--external">Get started with Charmed OSM</a></p>
+      <p>
+        <a href="/contact-us" class="p-button--positive">Get in touch</a>
+      </p>
+      <p>
+        <a href="https://tutorials.ubuntu.com/tutorial/charmed-osm-get-started" class="p-link--inverted p-link--external">Get started with Charmed OSM</a>
```

Comment:
```suggestion
        <a href="https://juju.is/tutorials/charmed-osm-get-started" class="p-link--inverted p-link--external">Get started with Charmed OSM</a>
```

This has been redirected... this is the final link.  The copy doc needs updating as well.